core-skills/web-application-development-with-react/README.md

```diff
@@ -33,6 +33,12 @@ Each section of the mark scheme will be separated into their respective categori
 Example answers for the questions be found on the hidden marking guide, however we do not require exact matches. It 
 is up to the assessor to determine whether or not the candidate has adequetly answered the question. 
 
+### The Application
+
+Build an app which communicates with an API, either one you write your own (you dont need to implement the API)
```

Comment:
Want to clarify, did we expect this to be part of Fox?

Reply:
Yeah we should clarify that's overall

Something like:

"By the end of this core skill you will have built an application which communicates with an API"

I'm thinking we write the API as a crossover with the TDD Dragon Core skill, as we need one there. So maybe we should make an "Chirp" API that we use for both projects